src/app/services/theme/theme.service.ts

```diff
@@ -10,7 +10,7 @@ export const DefaultTheme = {
   label: 'iX Dark',
   labelSwatch: 'blue',
   description: 'TrueNAS 12 default theme',
-  accentColors: ['blue', 'magenta', 'orange', 'cyan', 'yellow', 'violet', 'red', 'green'],
+  accentColors: ['blue', 'magenta', 'orange', 'cyan', 'yellow', 'violet', 'red', 'green', 'grey2'],
```

Comment:
Let's not add to the accentColors palette since those are used for chart legend colors among other things. The theme service actually adds some dynamic colors that aren't in the theme object. One of them is "grey" so you can actually use the `var(--grey)` css property instead